include/seastar/net/api.hh

```diff
@@ -389,13 +389,28 @@ public:
 
 /// @}
 
+/// Options for creating a listening socket. These options only have an effect
+/// when using the POSIX stack: all options are ignored on the native stack.
 struct listen_options {
     bool reuse_address = false;
     server_socket::load_balancing_algorithm lba = server_socket::load_balancing_algorithm::default_;
     transport proto = transport::TCP;
     int listen_backlog = 100;
     unsigned fixed_cpu = 0u;
     std::optional<file_permissions> unix_domain_socket_permissions;
+
+    /// If set, the SO_SNDBUF size will be set to the given value on the listening socket
+    /// via setsockopt. This buffer size is inherited by the sockets returned by
+    /// accept and is the preferred way to set the buffer size for this sockets since
```

Comment:
typo: should be "these" sockets.

Reply:
Same typo in the copy-pasted comment below.

Reply:
Eagle eye :eagle: ! I've fixed this in push https://github.com/scylladb/seastar/commit/5b33a115045eeb037ff41db9c22c213e45808508.